ext/popup.html

```diff
@@ -45,6 +45,12 @@
                         </div>
                     </div>
 
+                    <div id="no-dictionaries" hidden>
+                        <div class="entry">
+                            <p>No dictionaries have been installed or enabled yet.</p>
+                            <p><a href="#">Go to Dictionaries settings.</a></p>
```

Comment:
This url doesn't seem correct?

Reply:
Tapping the href which links to dictionary section in the Settings page will break the popup. I'm trying to find how to override this behavior.

I'm not sure if this popup ever shows up if user does not have any dictionary.

Reply:
> I'm not sure if this popup ever shows up if user does not have any dictionary.
<img width="406" alt="image" src="https://github.com/user-attachments/assets/e6446c9a-f45c-405d-b10d-09ddf76f2d56">

it's possible by doing a lookup via the context menu but it's kind of rare

Reply:
Just checked it again. It only breaks when I don't put target="__blank".